grails-app/controllers/au/org/ala/merit/OrganisationController.groovy

```diff
@@ -450,20 +459,87 @@ class OrganisationController {
 
     }
 
-    def viewOrganisationReport(String reportId) {
-        viewOrEditOrganisationReport(reportId, false)
+
+    def viewOrganisationReport(OrganisationReportCommand cmd) {
+        if (cmd.hasErrors()) {
+            error(cmd.errors.toString())
+            return
+        }
+
+        if (cmd.report.type == ReportService.PERFORMANCE_MANAGEMENT_REPORT) {
+            viewOrEditOrganisationReport(cmd.report, false)
+        }
+        else {
+            render model:cmd.model, view:'/activity/activityReportView'
+        }
+    }
+
+
```

Comment:
There is an editReport(String, String) and an editOrganisationReport(OrganisationReportCommand) method.  I suspect one of these is not used (presumably the former).